qa/L0_tf_multi_stream/test.sh

```diff
@@ -0,0 +1,124 @@
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+CLIENT_LOG_BASE="./client"
+
+DATADIR=`pwd`/models
+MODEL_SRCDIR=/data/inferenceserver/qa_custom_ops
+MODEL="graphdef_busyop"
+
+SERVER=/opt/tensorrtserver/bin/trtserver
+# Allow more time to exit. Ensemble brings in too many models
+SERVER_LOG_BASE="./inference_server.log"
+
+CLIENT_LOG="./client.log"
+MULTI_STREAM_CLIENT=multi_stream_client.py
+
+NUM_GPUS=${NUM_GPUS:=1}
+TOTAL_MEM=${TOTAL_MEM:=10000}
+source ../common/util.sh
+
+# A standard TITAN V does 1200 MHz. So this value
+# allows the busy loop kernel to run for over
+# 2 seconds. Note: This is close to the limit for
+# ints. The input to the busyloop model is an int
+NUM_DELAY_CYCLES=${NUM_DELAY_CYCLES:=2100000000}
+
+rm -f $SERVER_LOG_BASE* $CLIENT_LOG_BASE*
+
+export LD_PRELOAD=/data/inferenceserver/qa_custom_ops/libbusyop.so
+export CUDA_VISIBLE_DEVICES=$(seq -s, 0 $(( NUM_GPUS - 1 )))
+
+for INSTANCE_CNT in 2 4 8; do
+    # Create local model repository
+    rm -fr models && \
+        mkdir models && \
+        cp -r ${MODEL_SRCDIR}/*busyop* models/
+
+    # Establish baseline
+    echo "instance_group [ { kind: KIND_GPU, count: ${INSTANCE_CNT} } ]" >> models/${MODEL}/config.pbtxt
+    SERVER_ARGS="--model-store=${DATADIR} --exit-timeout-secs=120"
+    run_server
+    if [ "$SERVER_PID" == "0" ]; then
+        echo -e "\n***\n*** Failed to start $SERVER\n***"
+        cat $SERVER_LOG
+        exit 1
+    fi
+
+    # The first run of the client warms up TF/CUDA
+    set +e
+    python $MULTI_STREAM_CLIENT -v -i grpc -u localhost:8001 -m $MODEL -c $INSTANCE_CNT -n $NUM_DELAY_CYCLES >> /dev/null
```

Comment:
Don't use -n.
-c only has meaning with -d, you want to use -t. You should verify that perf_client is doing the right thing